conda/recipes/ucx-py/conda_build_config.yaml

```diff
@@ -1,12 +1,7 @@
 c_compiler_version:
   - 9
-
 cxx_compiler_version:
   - 9
 
-cython:
-  - " >=0.29.14,<3.0.0a0"
-pynvml:
-  - " >=11.4.1"
```

Comment:
We have moved this version constraint to `pyproject.toml` where it is now used for both conda & wheel packages